src/dune_rules/lib_info.mli

```diff
@@ -150,6 +150,8 @@ val special_builtin_support : _ t -> Special_builtin_support.t option
 
 val modes : _ t -> Lib_mode.Map.Set.t
 
+val modules : _ t -> Modules.t option
```

Comment:
This should return `Modules.t Source.t` for clarity.

Reply:
done here [8375c33](https://github.com/ocaml/dune/pull/6605/commits/8375c33f2ccc9ea1680a1b76e3c4323bbb6a0fe3)